### ISSUE TITLE
[Feat] Updated with relative time period

### ISSUE THREAD

**Issue by davipt** [vostpt/greve-transportes-perigosos#182]
If it makes sense to you, relative time periods are more helpful to understand "last updated", rather than full dates, as in this case the idea is to see how stale the information is.

So here's a suggestion for a quick timediff:

```
diff --git a/resources/js/map.js b/resources/js/map.js
index cd4a100..210a8dd 100644
--- a/resources/js/map.js
+++ b/resources/js/map.js
@@ -28,6 +28,19 @@ String.prototype.capitalize = function () {
     });
 };
 
+function timediff(date) {
+    let diffSeconds = Math.floor((new Date() - date) / 1000);
+    let days = Math.floor(diffSeconds / 86400);
+    let hours = Math.floor((diffSeconds % 86400) / 3600);
+    // ceil because we don't care about seconds
+    let minutes = Math.min(59, Math.ceil((diffSeconds % 3600) / 60));
+    var res = ""
+    if (res || days > 0) { res += " " + days + "d"; }
+    if (res || hours > 0) { res += " " + hours + "h"; }
+    if (res || minutes > 0) { res += " " + minutes + "m"; }
+    return res.trim();
+}
+
 function loadPoints() {
     return new Promise(function (resolve, reject) {
         points = [];
@@ -131,9 +144,11 @@ function loadPoints() {
                 else {
                     tooltip += '<p style="margin-bottom:0">Marca: '+ fuelStation.brand +'</p>';
                 }
-                var date = new Date(fuelStation.updated_at.replace(/ /, 'T') + '+01:00');
+                let date = new Date(fuelStation.updated_at.replace(/ /, 'T') + '+01:00');
+                // let dateString = date.toLocaleString();
+                let dateString = timediff(date);
                 tooltip += '<p style="margin-bottom:0">ID: '+fuelStation.id+'</p>';
-                tooltip += '<p style="margin-bottom:0">Última atualização em: '+date.toLocaleString()+'</p>';
+                tooltip += '<p style="margin-bottom:0">Última atualização: '+dateString+'</p>';
                 points.push({
                     "type": "Feature",
                     "geometry": {
```

**Comment by miguelantoniosantos** [comment]
Implemented on #184 and #186.

Can you confirm this?

**Comment by davipt** [comment]
Perfect. Thanks
